question-servers/elements/pl_symbolic_input.py

```diff
@@ -1,10 +1,10 @@
 import lxml.html
 from html import escape
 import chevron
-import to_precision
 import prairielearn as pl
 import sympy
 import random
+import ast
 
 def prepare(element_html, element_index, data):
```

Comment:
Would it be a good idea to parse the `correct_answer` with the `evaluate()` function, to make sure that what the instructor provided is actually something that the student will be able to enter? For example, that the instructor didn't use a function that isn't available to the student.

Reply:
I agree. Addressed in next commit.